README.md

```diff
@@ -130,6 +131,13 @@ Set the variable ``PMA_ABSOLUTE_URI`` to the fully-qualified path (``https://pma
 * ``UPLOAD_LIMIT`` - if set, will override the default value for apache and php-fpm (default value is 2048 kb)
 * ``PMA_CONFIG_BASE64`` - if set, will override the default config.inc.php with the base64 decoded contents of the variable
 * ``PMA_USER_CONFIG_BASE64`` - if set, will override the default config.user.inc.php with the base64 decoded contents of the variable
+* ``PMA_ABSOLUTE_URI`` - the full URL to phpMyAdmin. Sometimes needed when used in a reverse-proxy configuration. Don't set this unless needed
+* ``PMA_CONTROLHOST`` - when set, this points to an alternate database host used for storing the phpMyAdmin Configuration Storage database
```

Comment:
```suggestion
* ``PMA_CONTROLHOST`` - when set, this points to an alternate database host used for storing the [phpMyAdmin Configuration Storage database](https://docs.phpmyadmin.net/en/latest/setup.html#phpmyadmin-configuration-storage)
```

Maybe a link to our docs could be a nice to have ?

Reply:
Excellent suggestion. I forgot this was Markdown for a moment and that linking is easy. I'll do this.